server/pageRenderer.js

```diff
@@ -43,25 +27,32 @@ function retrieveAssets() {
 const dllPlugin = __DEV__ ? '<script src="/vendors.dll.js"></script>' : '';
 
 export type PageRendererProps = {
-  body: string,
+  app: ?React$Element<*>,
   state: State | {||},
   helmet: *,
 };
 
 export default function pageRenderer({
-  body = '',
+  app = undefined,
   state = {},
   helmet,
 }: PageRendererProps = {}): string {
-  const { scripts, styles } = retrieveAssets();
-  const isSSR = body === '' ? 'false' : 'true';
+  const extractor = new ChunkExtractor({
+    statsFile: path.join(webpackClient.outputPath, 'loadable-stats.json'),
+    entrypoints: ['app'],
+  });
+  const collectedApp = extractor.collectChunks(app);
+
+  const { scripts, styles, links } = retrieveAssets(extractor);
+  const isSSR = app === undefined ? 'false' : 'true';
```

Comment:
I don't understand why `isSSR` must be a string, care to explain? If not, perhaps do something like
```
const isSSR = app !== undefined;
```

Reply:
If you see below, it's templated into the html. I guess it should work with a string :thinking:. But it doesn't really matter :shrug: